src/main/java/edu/hm/hafner/analysis/parser/MsBuildParser.java

```diff
@@ -20,13 +20,12 @@
  */
 public class MsBuildParser extends LookaheadParser {
     private static final long serialVersionUID = -2141974437420906595L;
-
-    private static final String MS_BUILD_WARNING_PATTERN
-            = "(?:^(?:.*)Command line warning ([A-Za-z0-9]+):\\s*(.*)\\s*\\[(.*)\\])|"
-            + ANT_TASK + "(?:(?:\\s*(?:\\d+|\\d+:\\d+)>)?(?:(?:(?:(.*?)\\((\\d*)(?:,(\\d+))?[a-zA-Z]*?\\)|.*LINK)\\s*:|"
-            + "(.*):)\\s*([A-z-_]*\\s(?:[Nn]ote|[Ii]nfo|[Ww]arning|(?:fatal\\s*)?[Ee]rror))[^A-Za-z0-9]\\s*:?\\s*([A-Za-z0-9\\-_]+)?"
-            + "\\s*:\\s(?:\\s*([A-Za-z0-9.]+)\\s*:)?\\s*(.*?)(?: \\[([^\\]]*)[/\\\\][^\\]\\\\]+\\])?"
-            + "|(.*)\\s*:.*error|\\s*warning\\s*(LNK[0-9]+):\\s*(.*)))$";
+        private static final String MS_BUILD_WARNING_PATTERN
+            = "(?:^(?:.*)Command line warning (?<a1>[A-Za-z0-9]+):\\s*(?<a2>.*)\\s*\\[(?<a3>.*)\\])|"
+            + ANT_TASK + "(?:(?:\\s*(?:\\d+|\\d+:\\d+)>)?(?:(?:(?:(?<a4>.*?)\\((?<a5>\\d*)(?:,(?<a6>\\d+))?[a-zA-Z]*?\\)|.*LINK)\\s*:|"
+            + "(?<a7>.*):)\\s*(?<a8>[A-z-_]*\\s(?:[Nn]ote|[Ii]nfo|[Ww]arning|(?:fatal\\s*)?[Ee]rror))[^A-Za-z0-9]\\s*:?\\s*(?<a9>[A-Za-z0-9\\-_]+)?"
+            + "\\s*:\\s(?:\\s*(?<a10>[A-Za-z0-9.]+)\\s*:)?\\s*(?<a11>.*?)(?: \\[(?<a12>[^\\]]*)[/\\\\][^\\]\\\\]+\\])?"
+            + "|(?<a13>.*)\\s*:.*(?<a14>:error|warning)\\s*(?<a15>LNK[0-9]+):\\s*(?<a16>.*)))$";
```

Comment:
The bug is still not resolved. 

That is why is I did not assert all the statements with their severity.